downloads/swan-lake-release-notes/swan-lake-alpha4/swan-lake-alpha4-release-note.md

```diff
@@ -333,17 +354,9 @@ To view bug fixes, see the GitHub milestone for Swan Lake <VERSION> of the repos
 
 ### Developer Tools Updates
 
-#### Language Server 
-
-To view bug fixes, see the [GitHub milestone for Swan Lake <VERSION>](https://github.com/ballerina-platform/ballerina-lang/issues?q=is%3Aissue+is%3Aclosed+milestone%3A%22Ballerina+Swan+Lake+-+Alpha4%22+label%3AType%2FBug+label%3ATeam%2FLanguageServer).
```

Comment:
Why this is removed? It seems we have one issue as fixed

Reply:
@nadeeshaan @prabushi - What is the diff between the 2 labels "Team/LanguageServer" and "Team/DevTools"? Do we have to include both or will the DevTools label filter include LanguageServer related issues as well?

Reply:
@praneesha DevTools are excluding the Language Server issues. Hence we need to track both of them separately, due to this reason we have added the `Team` prefix for the labels.

Reply:
Added it back. Thanks a lot!